book/modules/appendix1.tex

```diff
@@ -351,24 +388,189 @@
 	\end{enumerate}
 
 	\begin{example}
-		XXX Finish. Unique solution to a 3x3 system with no row swaps; Use back subsitution
+		Find a solution to the following system:
+		\[
+			\systeme[abc]{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}.
+		\]
+		To do so, we rewrite the system as an augmented matrix then row reduce.
+		\begin{align*}
+			{\footnotesize
+			\systeme[abc]{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}
+			} \rightarrow
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				2&7&5 & 2\\
+				-1&-4&0 & 11
+			\end{bmatrix}
+			&\xrightarrow{\text{row}_2\mapsto\text{row}_2-2\text{row}_1}
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				0&1&1 & 0\\
+				-1&-4&0 & 11
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3+\text{row}_1+\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				0&1&1 & 0\\
+				0&0&3 & 12
+			\end{bmatrix}
+			\rightarrow
+			{\footnotesize
+				\systeme[abc]{
+				a+3b+2c=1,
+				b+c=0,
+				3c=12
+				}
+			}
+		\end{align*}
+		The third row reveals that $c=4$ and we substitute that into the second row to find $b=-4$. Now we can substitute $b=-4$ and $c=4$ into the first row and we obtain $a=5$.
+		
+		As a vector, the solution is $\mat{5\\-4\\4}$ and since it is unique, the solution set is $\Set*{\mat{5\\-4\\4}}$.
```

Comment:
-> The solution is `(a,b,c)=(5,-4,4)`. Since this is the only solution to the system, the solution set is ...